py/server/deephaven/parquet.py

```diff
@@ -144,9 +152,12 @@ def read(
             inferred.
         table_definition (Union[Dict[str, DType], List[Column], None]): the table definition, by default None. When None,
             the definition is inferred from the parquet file(s). Setting a definition guarantees the returned table will
-            have that definition. This is useful for bootstrapping purposes when the initial partitioned directory is
+            have that definition. This is useful for bootstrapping purposes when the initially partitioned directory is
             empty and is_refreshing=True. It is also useful for specifying a subset of the parquet definition. When set,
             file_layout must also be set.
+        special_instructions (Optional[object]): Special instructions for reading parquet files, useful when reading
+            files from a non-local file system, like S3. By default, None. When
```

Comment:
Fragment.

Reply:
I didn't fully get what you mean. Were you pointing out the "When" at the end? I have removed that.